app/controllers/providers/capital_assessment_results_controller.rb

```diff
@@ -9,6 +9,7 @@ def show
     end
 
     def update
+      LegalFramework::MeritsTasksService.call(legal_aid_application)
```

Comment:
All lgtm.

Should there be a TODO or a note somewhere to remove this when the MP flag is switched on? As I see the merits task list controller will then handle it.

Reply:
I was tempted to leave both in... the MeritsTaskListController will create one if needed and this one somehow fails!